packages/cra-template-safe-app/template/src/setupProxy.js

```diff
@@ -2,14 +2,13 @@
 // https://create-react-app.dev/docs/proxying-api-requests-in-development/#configuring-the-proxy-manually
 
 module.exports = function (app) {
-  app.use("/manifest.json", function (req, res, next) {
+  app.use('/manifest.json', function (req, res, next) {
```

Comment:
If we only have the cors proxy for the manifest can the icons be properly loaded?

Reply:
Good catch. No, they can't. I'll just apply it for every route

Reply:
Hmm, thinking about it a little bit more I'm not sure. The icons can be properly loaded via HTML `img` tag. They cannot be fetched from js code, though.

The interface doesn't display it because it tries to fetch it first: https://github.com/gnosis/safe-react/blob/development/src/routes/safe/components/Apps/utils.ts#L266 and if it doesn't work it doesn't try to use the URL anymore and uses placeholder image

Reply:
:facepalm: ... yeah not optimal not sure ... 

Little bit different question shouldn't react app rewired also cover this?

Reply:
It's not well maintained and already has some caveats https://github.com/gnosis/safe-react-apps/issues/88

Reply:
If we can stay with "react-scripts" it's better to stay with it, chances of getting updates are way higher

Reply:
Makes sense :)